packages/tailwindcss-language-service/src/documentLinksProvider.ts

```diff
@@ -7,6 +7,7 @@ import { findAll, indexToPosition } from './util/find'
 import { getTextWithoutComments } from './util/doc'
 import { absoluteRange } from './util/absoluteRange'
 import * as semver from './util/semver'
+import { getCssBlocks } from './util/language-blocks'
```

Comment:
Can't edit non-changed lines. But the imported `isCssDoc`, `getLanguageBoundaries` and `getTextWithoutComments` are unused.